notifier/events/custom_capture_cc_event.py

```diff
@@ -0,0 +1,23 @@
+from django.conf import settings
+from notifier.event_handler.event import Event
+
+
+class CustomCaptureCCEvent(Event):
+
+    def __init__(self, job_group, assay):
+        self.job_group = job_group
+        self.assay = assay
+
+    @classmethod
+    def get_type(cls):
+        return "CustomCaptureCCEvent"
+
+    @classmethod
+    def get_method(cls):
+        return "process_custom_capture_cc_event"
+
+    def __str__(self):
+        TEMPLATE = """
+        Ticket in AdminHold because of the assay: {assay}. {cc}
```

Comment:
I think maybe it should be CustomCaptureCCEvent instead of AdminHold event

Reply:
The JIRA states are fine

Reply:
It can be more assays then CustomCapture that can put ticket to admin hold. But we will put assay in comment as well.